gulpfile.babel.js

```diff
@@ -267,7 +267,6 @@ gulp.task('metadata', () => {
 gulp.task('default', ['clean'], cb => {
   runSequence(
     ['styles', 'styles-grid'],
-    ['scripts'],
```

Comment:
Well, we need to test the script when building, so Mocha runs. Mocha forces `scripts` to run since it is testing them it wants it all fresh. If we run `scripts` before mocha, we are just creating 2 script cycles instead of the one (mocha) that  will do it all on its own. This helps us keep the total number of task runs down so we aren't building scripts and styles numerous times like we were before.